processor/k8sattributesprocessor/options.go

```diff
@@ -50,6 +50,9 @@ func withPassthrough() option {
 // enabledAttributes returns the list of resource attributes enabled by default.
 func enabledAttributes() (attributes []string) {
 	defaultConfig := metadata.DefaultResourceAttributesConfig()
+	if defaultConfig.K8sClusterUID.Enabled {
+		attributes = append(attributes, "k8s.cluster.uid")
```

Comment:
There are two references to `k8s.cluster.uid` in this file (here and line 173). Ideally, we would use `conventions.AttributeK8SClusterUID`, but the collector semantic conventions are not up to date. In the meantime can we replace those references with a constant, and put a TODO above the constant to replace it when semconv v.1.21.0 is available?

Reply:
Yes, was thinking to do it, but then would need to update semconv and bump the deps here.
Thanks for suggested adding a TODO, done!!